apps/admin/src/containers/BrokerCAD/Clients/ClientSelect.tsx

```diff
@@ -1,21 +1,24 @@
 import React, { useCallback, useEffect, useState } from "react";
-
 import { DropdownProps, Select } from "semantic-ui-react";
 import { Client } from "./Client";
 import { useFxRates } from "@thecointech/shared/containers/FxRate";
-import { UserState } from "./types";
-import { getAllUserData } from "./data";
+import { AllDataArray, getAllUserData } from "./data";
+import { AccountMap } from '@thecointech/shared/containers/AccountMap';
 
 export const ClientSelect = () => {
 
-  const [users, setUsers] = useState([] as UserState[]);
+  const [users, setUsers] = useState<AllDataArray>([]);
```

Comment:
AllDataArray is that the new any?

Reply:
hahaha!!! I could name that better, couldn't I?

Reply:
done in next PR